src/AccessibilityInsights.SharedUx/Controls/ScannerResultControl.xaml.cs

```diff
@@ -37,6 +37,7 @@ public ScannerResultControl()
             InitializeComponent();
             _list = new List<ScanListViewItemViewModel>();
             Resources.Source = new Uri(@"pack://application:,,,/AccessibilityInsights.SharedUx;component/Resources/Styles.xaml", UriKind.Absolute);
+            this.spHowToFix.DataContext = null;
```

Comment:
No need for explicit `this`:

```suggestion
            spHowToFix.DataContext = null;
```